### PR TITLE
Wazuh indexer and dashboard for 4.3

### PR DESCRIPTION
**Pull request by teddytpc1**
This PR closes https://github.com/wazuh/wazuh-automation/issues/732.
It contains the changes to deploy Wazuh indexer and Dashboard.

### CODE REVIEW

File: playbooks/wazuh-opensearch-production-ready.yml [hunk 1]
```diff
@@ -0,0 +1,212 @@
+---
```
Comment (alberpilot) [inline playbooks/wazuh-opensearch-production-ready.yml:1]: Can we change this file name by `wazuh-indexer-production-ready.yml`?

**Review by alberpilot — CHANGES_REQUESTED** [review]
I see too many references to opensearch instead of indexer. Please make the changes to obtain indexer-related naming.

File: playbooks/wazuh-opensearch-production-ready.yml (hunk 1 above)
Comment (teddytpc1) [reply playbooks/wazuh-opensearch-production-ready.yml:1]: yes

Comment (teddytpc1) [reply playbooks/wazuh-opensearch-production-ready.yml]: This was changed to `wazuh-production-ready.yml`.

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**